packages/frontend/src/hooks/useCartesianChartConfig.ts

```diff
@@ -403,78 +393,90 @@ const useCartesianChartConfig = ({
 
     // Generate expected series
     useEffect(() => {
-        const areaStyleConfig = areaStyle ? {} : undefined;
-
         if (isCompleteLayout(dirtyLayout) && resultsData) {
-            let expectedSeriesMap: Record<string, Series>;
```

Comment:
Moved all this logic to be inside the `setDirtyEchartsConfig` so we can know the `defaultCartesianType` and  `defaultAreaStyle`